numpy/core/tests/test_umath.py

```diff
@@ -3157,6 +3157,13 @@ def test_rint_big_int():
     # Rint should not change the value
     assert_equal(val, np.rint(val))
 
+@pytest.mark.parametrize('ftype', [np.float32, np.float64])
+def test_maximum_accumulate(ftype):
+    arr = np.array([0.61,0.60,0.77,0.41,0.19], dtype=ftype)
+    out_max = np.array([0.61,0.61,0.77,0.77,0.77], dtype=ftype)
+    out_min = np.array([0.61,0.60,0.60,0.41,0.19], dtype=ftype)
+    assert_equal(np.maximum.accumulate(arr), out_max)
+    assert_equal(np.minimum.accumulate(arr), out_min)
```

Comment:
Doesn't this only check the non-overlapping case? If so, can you rename it to `test_overlapping_accumulate` and add a case that checks that, I think the original issue provides an instance. Also formatting of the values needs a `' '` after the ','.

Reply:
`np.maximum.accumulate(arr)` seems to always call `np.maximum` with overlapping arrays. These tests fail the master branch (just like the one provided in the original issue). I fixed the formatting and renamed the test.